edx_when/api.py

```diff
@@ -26,15 +25,27 @@
 FIELDS_TO_EXTRACT = ('due', 'start', 'end')
 
 
-def _content_dates_cache_key(course_key, query_dict):
-    """Memcached key for ContentDates given course_key and filter args."""
+def _content_dates_cache_key(course_key, query_dict, subsection_and_higher_only, published_version):
```

Comment:
I'm not in 😍 with the name `subsection_or_higher_only` either - but it's the best I could think of and it's at least explicit without mentioning outlines. I agree with your proposed ideal solution as well. But I pulled back from that solution (passing in a list of block types) to avoid the extra scope & testing - and KISS/YAGNI.

Reply:
Yeah, that's fair. Thank you!